Package.swift

```diff
@@ -58,6 +59,7 @@ let package = Package(
         .product(name: "AsyncHTTPClient", package: "async-http-client"),
         .product(name: "SwiftToolsSupport-auto", package: "swift-tools-support-core"),
         openCombineProduct,
+        "Splash",
```

Comment:
Could this be the reason build [fails on 5.2](https://github.com/swiftwasm/carton/pull/122/checks?check_run_id=1218022440#step:3:1250), needing a full product line like the others?

@MaxDesiatov why do we still need the macOS 5.2 check? Xcode 12 & 5.3 are plenty stable by now.

Reply:
It could be, but I don't think it is, you need to specify the package the product belongs to if the name of the product is not the same as the name of the package.

Xcode 12 and 5.3 are stable enough, but I want to support old Xcode versions for as long as possible, so that there's little friction for people trying out SwiftWasm who aren't on the latest and greatest yet. Checking with old Xcode on CI allows us to check all works fine for them.